mathesar_ui/src/stores/table-data/constraints.ts

```diff
@@ -23,13 +32,29 @@ export interface ConstraintsData {
   constraints: Constraint[],
 }
 
+function api(url: string) {
+  return {
+    get() {
+      return getAPI<PaginatedResponse<Constraint>>(`${url}?limit=500`);
+    },
+    add(constraintDetails: Partial<Constraint>) {
+      return postAPI<Partial<Constraint>>(url, constraintDetails);
+    },
+    remove(constraintId: Constraint['id']) {
+      return deleteAPI(`${url}${constraintId}`);
```

Comment:
Same as my previous comment. We can add a trailing slash here.

Reply:
Addressed in 7d3ba58d38a291cabed95322d59bbd8e26532c8b